src/components/timeline/GlobalTrack.js

```diff
@@ -101,6 +119,17 @@ class GlobalTrackComponent extends PureComponent<Props> {
       return;
     }
 
+    if (
+      // Is this a keypress?
+      typeof event.key === 'string' &&
+      // Only allow Spacebar and Enter, which signals the button is being pressed.
+      event.key !== ' ' &&
+      event.key !== 'Enter'
+    ) {
+      // Ignore this keypress.
+      return;
+    }
+
     const { selectTrack, trackReference } = this.props;
     selectTrack(trackReference, getTrackSelectionModifier(event));
```

Comment:
Hmm.. I'm going to skip this one as optional. I went back and forth on this a few times, especially with the code duplication. Right now there are 2 copies of this code, and this would make 4 essentially.